src/Module.cpp

```diff
@@ -846,6 +846,7 @@ void compile_multitarget(const std::string &fn_name,
             Target::MSAN,
             Target::NoRuntime,
             Target::TSAN,
+            Target::SANCOV,
```

Comment:
annoying style nit: all-caps `SANCOV` seems out of place here as an all-caps abbreviation, despite MSAN, etc, since the latter are commonly referred to that way in LLVM docs but the former seems to be `SanitizerCoverage`. If we are using this only for testing purposes then I think we should use either `SanitizerCoverage` or `sanitizer_coverage` to fit in with existing patterns. (Tagging @abadams here for his opinion as well.)

Reply:
IMO SanitizerCoverage to match the llvm docs, but I don't feel strongly about it

Reply:
Do you want me to change all instances of SANCOV to SanitizerCoverage, or just some?

> If we are using this only for testing purposes

I'm not sure what you mean by that? Roughly, the feature will be requested the way it is requested in the test.

Reply:
> change all instances of SANCOV to SanitizerCoverage, or just some?

I'd vote for all (except the ones in HalideRuntime.h, which should be `sanitizer_coverage` to match the conventions there)

Reply:
Ok, let me know if i did it wrong.